src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -47,14 +54,64 @@ const InfoTile = ({ title, children }) => {
   ])
 }
 
+class EditWorkspaceModal extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      description: props.workspace.workspace.attributes.description || '',
```

Comment:
Why not have this take `workspace.workspace` as its prop, rather than `workspace`? Save a level? Or just `namespace`, `name`, `description`? Make it structure agnostic?

Reply:
The original thought was that this would be a general "edit workspace" dialog. Obviously it didn't end up that way, at least not right now.

Also, the workspace object is a fairly coherent "type" in our ecosystem, so it seemed like a reasonable way to get all the relevant info into the modal. I'm inclined to ponder it for a bit, and perhaps revisit the decision later.